toolchains/v100.persee/debug_toolchain.cmake

```diff
@@ -5,14 +5,10 @@ include(${CMAKE_CURRENT_LIST_DIR}/../cicd_default_toolchain.cmake)
 set(CMAKE_BUILD_TYPE Debug)
 
 # Compiler options
-set(CMAKE_CXX_COMPILER ${CMAKE_CURRENT_LIST_DIR}/../../vendor/kokkos/bin/nvcc_wrapper)
+set(CMAKE_CXX_COMPILER nvcc_wrapper)
+set(CMAKE_CXX_EXTENSIONS OFF) # Avoid a Kokkos warning that will force if to OFF anyway when compiling with nvcc
```

Comment:
Should `set(CMAKE_CXX_EXTENSIONS OFF)` be moved to `toolchains/cicd_default_toolchain.cmake` ? Or is it ON for CPU configs?

Reply:
Kokkos does not force it to `OFF` for any other compiler than `nvcc`. So I would keep it only for Nvidia GPU clusters using `nvcc`.